bin/oneflow

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../build/lib');
+require('../build/src/lib');
```

Comment:
hm, what caused this to change? The tsconfig updates?

Reply:
This is due to the `resolveJsonModule` change introduced on typescript 3.0. Now `tsc` knows how to adapt the reference to json files on the build folder but it has to have the same depth as in the original root folder. You can check this by making a build and looking at the json files and their paths.

Reply:
Thanks for your reply.